.github/workflows/gerrit-compose-required-maven-verify.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
           phases: clean, deploy
           settings-file: settings.xml
           # yamllint disable-line rule:line-length
-          maven-opts: --show-version --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.repo.local=/tmp/r -Dorg.ops4j.pax.url.mvn.localRepository=/tmp/r -DaltDeploymentRepository=staging::default::file:"${GITHUB_WORKSPACE}"/m2repo
+          maven-opts: -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.repo.local=/tmp/r -Dorg.ops4j.pax.url.mvn.localRepository=/tmp/r -DaltDeploymentRepository=staging::default::file:"${GITHUB_WORKSPACE}"/m2repo
```

Comment:
Can you please break this up using `>-` so it's a lot easier to read? See https://yaml-multiline.info/ for the different string folding styles

Reply:
Sure